src/common/low_precision_transformations/src/fold_fake_quantize.cpp

```diff
@@ -18,7 +19,10 @@ namespace low_precision {
 
 FoldFakeQuantizeTransformation::FoldFakeQuantizeTransformation(const Params& params) : LayerTransformation(params) {
     MATCHER_SCOPE(FoldFakeQuantizeTransformation);
-    auto fakeQuantize = pattern::wrap_type<opset1::FakeQuantize>();
+    auto matcher = std::make_shared<pattern::op::Or>(OutputVector {
+            pattern::wrap_type<opset1::Multiply>(),
+            pattern::wrap_type<opset1::FakeQuantize>()
+        });
```

Comment:
We handle in this transformation completely different cases without any common parts: FakeQuantize on constant path or with equal constants, and Multiply with zero. I think this could be a good reason to move Multiply part to separate transformation (something like `FoldZeroMultiply`). @eshoguli what do you think?

Reply:
OK, separated

Reply:
Looks like we also need someone from `openvino-docs-maintainers` to review